test/lib/agent/update_package_message_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'katello_test_helper'
+
+module Katello
+  module Agent
+    class UpdatePackageMessageTest < ActiveSupport::TestCase
+      def test_all_package_update
+        json = UpdatePackageMessage.new(content: [], consumer_id: "100000").json
+        assert json[:request][:args][1][:all]
+        assert_empty json[:request][:args][0][0][:unit_key]
+      end
+
+      def test_content_package_update
+        json = UpdatePackageMessage.new(content: ["foo.rpm"], consumer_id: "100000").json
+        assert_nil json[:request][:args][1][:all]
```

Comment:
Works fine, but might be nice to use the same variable names as the corresponding code so it's a bit easier to see what's going on:
```suggestion
        options = json[:request][:args][1]
        units = json[:request][:args][0]
        assert_nil options[:all]
        assert_equal "foo.rpm", units[0][:unit_key][:name]
```